app/components/confirm-route-leave.js

```diff
@@ -29,6 +29,10 @@ export default class ConfirmRouteLeaveComponent extends Component {
       this.args.onCancel(transition);
     } else {
       transition.abort();
+
+      if (window.history) {
```

Comment:
Ideally we solve this using the router, perhaps using https://api.emberjs.com/ember/4.8/classes/EmberRouter/methods/replaceWith?anchor=replaceWith to revert to the old route?

Reply:
I tried using **relaceWith** router method, but for some reason was not working. I will try again today and will let you know.

Reply:
@nvdk replaceWith  is not available in ember 3.28, careful when you look at the api docs to select the right version

Reply:
hmm odd that it isn't offered, that seems like a bug :thinking:

Reply:
so yeah, tested it. the function is there, seems it somehow got dropped from the documentation

Reply:
that said, it seems non trivial to get this working with replaceWith, so I'd suggest to merge this as is

Reply:
I tried to get it working with **replaceWith**, but that is causing some weird behaviour.